scripts/zkai_jstprove_statement_envelope_benchmark.py

```
@@ -0,0 +1,646 @@
+    _check_artifact_hashes(envelope)
+    external_verify(envelope)
+
+
+def verify_proof_only(
+    envelope: dict[str, Any],
+    external_verify: Callable[[dict[str, Any]], None] = jstprove_verify,
+) -> None:
+    external_verify(envelope)
+
+
+def classify_error(message: str) -> str:
```

Ruff (0.15.12)

[warning] Too many return statements (8 > 6)

(PLR0911)